plugin/textobj/between.vim

```diff
@@ -12,12 +12,14 @@ let s:save_cpo = &cpo
 set cpo&vim
 
 " Interface  "{{{1
+let g:textobj_betweenimpl_no_default_key_mappings = 1
 call textobj#user#plugin('betweenimpl', {
 \      '-': {
 \        'select-a': 'af',  '*select-a-function*': 'textobj#between#impl_select_a',
 \        'select-i': 'if',  '*select-i-function*': 'textobj#between#impl_select_i',
```

Comment:
We can use empty default `{lhs}`.
```suggestion
\        'select-a': [],  '*select-a-function*': 'textobj#between#impl_select_a',
\        'select-i': [],  '*select-i-function*': 'textobj#between#impl_select_i',
```

Reply:
I cannot find such documentation in the help of textobj-user, but it seems to work and I think this implementation is simple and better than mine. I'll apply this change.

Reply:
You can find an example for defining textobj without default key-mappings in `:help textobj-user-example-filetype`.

Reply:
No way! It's completely my fault...
Thank you.